src/uncontrast/pages/GrandOverviewPie/GrandOverviewPie.jsx

```diff
@@ -330,12 +330,12 @@ export default function GrandOverviewPie() {
                   buildUrl(pageName, "min_number_of_experiments", e, navigate);
                 }}
               />
-              <SignificanceFilter
+              {/* <SignificanceFilter
```

Comment:
removed this on purpose?

Reply:
i was asked to by maor... but its commented because i suspect he wil want it back